query/language/insert.feature

```diff
@@ -219,7 +219,7 @@ Feature: TypeQL Insert Query
       """
     Given transaction commits
     Given connection open write transaction for database: typedb
-    When typeql write query; fails with a message containing: "Type-inference derived an empty-set for some variable"
+    When typeql write query; fails with a message containing: "Type-inference was unable to find compatible types for the pair of variables"
```

Comment:
This would be more meaningful if you added the variable names too in at least one of these, if they are user-written vars that have the error

Reply:
They're almost always `_anonymous`